nextflow_schema.json

```diff
@@ -44,6 +28,11 @@
                     "description": "The output directory where the results will be saved. You have to use absolute paths to storage on Cloud infrastructure.",
                     "fa_icon": "fas fa-folder-open"
                 },
+                "analysis": {
+                    "type": "string",
+                    "fa_icon": "fas fa-cog",
```

Comment:
I think it will complain if there's no description provided.

Reply:
solved, thanks for pointing this out, it was actually a bug on the `nf-core schema build` command